app/src/main/java/android/support/v17/leanback/supportleanbackshowcase/app/media/VideoConsumptionExampleFragment.java

```diff
@@ -60,40 +63,54 @@ protected void onRowChanged(PlaybackControlsRow row) {
         surface.getHolder().addCallback(new SurfaceHolder.Callback() {
             @Override
             public void surfaceCreated(SurfaceHolder holder) {
+                Log.d(TAG, "surfaceCreated called.");
```

Comment:
Fixed, Yes, setting display to null in surfaceDestroyed() fixed the crash when pressing play/pause button in gregarious mode.